packages/squiggle-lang/src/rescript/Reducer/Reducer_Type/Reducer_Type_TypeChecker.res

```diff
@@ -0,0 +1,81 @@
+//TODO: Work in progress. Code is commented to make an a release of other features
```

Comment:
Thanks for adding. It would be more useful if this comment was right above the commented-out code though, to be safe.